pages/index.js

```diff
@@ -39,40 +40,49 @@ export default function HomePage() {
   }
 
   return (
-    <QuoteContainer>
-      <StyledHeadline>
-        {moodMode ? moodMode : "Hello, how are you feeling today?"}
-      </StyledHeadline>
-      {moodMode ? (
-        <>
-          <StyledPostionRelative>
-            <StyledQuotationMarks
-              src={"/quote.svg"}
-              width={100}
-              height={100}
-              alt="quote decoration"
-            />
-          </StyledPostionRelative>
-          <QuoteText>{currentQuote.quote}</QuoteText>
-          <AuthorText>{currentQuote.author}</AuthorText>
-        </>
-      ) : (
-        <>
-          <form
-            onSubmit={(event) => handleMood(event, event.target.mood.value)}
-          >
-            <label htmlFor="mood">My mood today is...</label>
-            <br />
-            <br />
-            <span>😔</span>
-            <input type="range" min={1} max={10} id="mood" name="mood"></input>
-            <span>🤩</span>
-            <br />
-            <Button type="submit">Send</Button>
-          </form>
-        </>
-      )}
-    </QuoteContainer>
+    <>
+      <QuoteContainer>
+        <StyledHeadline>
+          {moodMode ? moodMode : "Hello, how are you feeling today?"}
+        </StyledHeadline>
+        {moodMode ? (
+          <>
+            <StyledPostionRelative>
+              <StyledQuotationMarks
+                src={"/quote.svg"}
+                width={100}
+                height={100}
+                alt="quote decoration"
+              />
+            </StyledPostionRelative>
+            <QuoteText>{currentQuote.quote}</QuoteText>
+            <AuthorText>{currentQuote.author}</AuthorText>
+          </>
+        ) : (
+          <>
+            <form
+              onSubmit={(event) => handleMood(event, event.target.mood.value)}
+            >
+              <label htmlFor="mood">My mood today is...</label>
+              <br />
+              <br />
+              <span>😔</span>
```

Comment:
I think this feature is really cool! Just a small tip: The emoji in the span element is already good. If you now give the span element an aria label, that's great for accessibility!
```suggestion
              <span aria-label="sad emoji">😔</span>
```